tests/bin/docker-compose.yml

```
@@ -0,0 +1,40 @@
+version: '2.4'
+services:
+    redis:
+        image: redis:alpine
+        command: [ redis-server, --maxmemory 128mb, --maxmemory-policy volatile-lru, --save "" ]
+
+    db:
+        image: mariadb:10.7
+        working_dir: /application
+        command: [mysqld, --character-set-server=utf8mb4, --collation-server=utf8mb4_unicode_ci, --innodb-file-per-table=1]
+        environment:
+            MYSQL_ROOT_PASSWORD: ROOT
+            MYSQL_DATABASE: pimcore_test
+            MYSQL_USER: pimcore
+            MYSQL_PASSWORD: pimcore
+
+    php-fpm:
+#        user: '1000:1000' # set to your uid:gid
+#        image: pimcore/pimcore:PHP8.1-fpm
+        build: .
+        environment:
+            PHP_IDE_CONFIG: "serverName=localhost"
+            COMPOSER_HOME: /var/www/html
+            PIMCORE_PROJECT_ROOT: /var/www/html
+            APP_ENV: test
+            PIMCORE_TEST: 1
+            PIMCORE_TEST_DB_DSN: "mysql://pimcore:pimcore@db:3306/pimcore_test"
+            PIMCORE_TEST_REDIS_DSN: "redis://redis:6379"
+        depends_on:
+            - db
+        volumes:
+            - ../../.:/var/www/html:cached
```

This is a legacy flag that's not necessary anymore (it does nothing in recent docker versions) and thus can be removed:

```suggestion
            - ../../.:/var/www/html
```